5-Heroku/server/public/main.1d8726c4f66e42fe6d55.js

```diff
@@ -0,0 +1 @@
-9a-f]{12}|00000000-0000-0000-0000-000000000000)$/i,gr=function(e){return"string"==typeof e&&yr.test(e)};for(var mr=[],vr=0;vr<256;++vr)mr.push((vr+256).toString(16).substr(1));const br=function(e,t,r){var n=(e=e||{}).random||(e.rng||pr)();if(n[6]=15&n[6]|64,n[8]=63&n[8]|128,t){r=r||0;for(var i=0;i<16;++i)t[r+i]=n[i];return t}return function(e){var t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:0,r=(mr[e[t+0]]+mr[e[t+1]]+mr[e[t+2]]+mr[e[t+3]]+"-"+mr[e[t+4]]+mr[e[t+5]]+"-"+mr[e[t+6]]+mr[e[t+7]]+"-"+mr[e[t+8]]+mr[e[t+9]]+"-"+mr[e[t+10]]+mr[e[t+11]]+mr[e[t+12]]+mr[e[t+13]]+mr[e[t+14]]+mr[e[t+15]]).toLowerCase();if(!gr(r))throw TypeError("Stringified UUID is invalid");return r}(n)};!function(){const e=(e,s)=>{const a=document.createElement("button"),u=`${e}-button`;return a.classList.add(u),a.classList.add("button-style"),a.appendChild(r(e,s)),a.addEventListener("click",(function(){!function(e,r){switch(e){case"delete":!async function(e){let r=document.getElementById(`todo-list-item-${e}`);await(async e=>t().delete(`${n}${e}`,{withCredentials:!0}).then((e=>e.data)).catch((e=>{console.log(e)})))(e),r.remove()}(r);break;case"edit":!async function(e){let t=document.getElementById(`todo-item-${e}`);const r=await i(e);r.todoText=t.value,await o(r),t.disabled=!t.disabled}(r)}}(e,s)})),a},r=(e,t)=>{const r=document.createElement("i");let n="",i="";switch(e){case"delete":n="fas fa-trash",i=`icon-delete-${t}`;break;case"edit":n="fas fa-edit",i=`icon-edit-${t}`;break;case"check":n="fa-check",i=`icon-check-${t}`}return r.className=n,r.id=i,r};function s(t){const r=document.getElementById("todo-list"),n=function(t){const{todoText:r,todoId:n}=t,s=document.createElement("li"),a=document.createElement("input");a.className=dr.todoItem,t.checked&&a.classList.toggle(dr.completed),a.disabled=!!t.checked,a.id=`todo-item-${n}`,a.value=r;const u=e("delete",n),l=e("edit",n),c=(e=>{const t=document.createElement("input");return t.type="checkbox",t.checked=e.checked,t.id=`checked-${e.todoId}`,t.className=dr.checked,t.classList.add("checked"),t.addEventListener("click",(function(){!async function(e){document.getElementById(`todo-item-${e}`).classList.toggle(dr.completed);const t=document.getElementById(`checked-${e}`).checked,r=await i(e);r.checked=t,await o(r)}(e.todoId)})),t})(t);return s.appendChild(c),s.appendChild(a),s.appendChild(u),s.appendChild(l),s.className=dr.todoListItem,s.id=`todo-list-item-${n}`,s}(t);r.appendChild(n)}!async function(){(await(async e=>t().get(`${n}/allTodos`,{withCredentials:!0}).then((e=>e.data)).catch((e=>{console.log(e)})))()).forEach((e=>{s(e)}))}(),document.getElementById("todo-button").addEventListener("click",(function(e){e.preventDefault();const r=document.getElementById("new-task-input");""!=r.value.trim()?async function(e){const r={todoText:e,todoId:br(),checked:!1};await(async e=>t().post(`${n}`,e,{withCredentials:!0}).catch((e=>{console.log(e)})))(r),s(r)}(r.value):alert("You didnt enter any Todo"),r.value=""}))}()})()})();
```

Comment:
what is this?

Reply:
this is the app it self after i build it so i can use it for the heroki

Reply:
what i thought.. check if you can do it on heroku itself. i'm pretty sure u don't need to commit your dist folder

Reply:
yea i didnt need to commit to this git here.